spec/models/cart_spec.rb

```diff
@@ -1,16 +1,37 @@
 require "rails_helper"
 
 RSpec.describe Cart, type: :model do
+  let!(:user) { User.create(email: "example@example.com", password: "password", name: "example") }
 
-  xcontext "with valid attributes" do
-    it "an item is added to the cart" do
-      item = create(:item)
-      cart = Cart.new(nil)
-      cart.add_item(item)
-      expect(cart.cart_items.count).to eq(1)
-      item2 = Item.create
-      cart.add_item(item2)
-      expect(cart.cart_items.count).to eq(2)
-    end
+  let!(:loan_request) { LoanRequest.create(title: "Farm Tools",
+                                           description: "help out with the farm tools",
+                                           amount: "100",
+                                           requested_by_date: "2015-06-01",
+                                           repayment_begin_date: "2015-12-01",
+                                           contributed: "50",
+                                           repayment_rate: "monthly",
+                                           user_id: user.id)
+  }
+
+  let!(:loan_request2) { LoanRequest.create(title: "Farm Stuff",
+                                            description: "help out with the farm tools",
+                                            amount: "600",
+                                            requested_by_date: "2015-09-21",
+                                            repayment_begin_date: "2016-10-01",
+                                            contributed: "25",
+                                            repayment_rate: "monthly",
+                                            user_id: user.id)
+    }
```

Comment:
`end` at 24, 4 is not aligned with `let!(:loan_request2) { LoanRequest.create(title: "Farm Stuff",` at 16, 2